PC Dev Suggestions

hmtinc opened the pull request:
### Zoom Issues
closes #198 - The padding of each overlay has been set to 0 and the graph is automatically fit after each layout run. This fixes all the jittering issues that were present in the beta 

### Tooltip  Issues
closes #190 - All comments that contain `REPLACED` have been filtered out.
closes #235 - Duplicate Cellular Locations has been removed from the tooltip 
closes #242 - Fixed a bug where a tooltip would not disappear when a complex is collapsed 

### Metadata Issues
closes #236 - Tooltips now appear on molecular interactions.

d2fong commented on src/client/features/view/components/menu/index.js:
```diff
@@ -51,6 +52,10 @@ class Menu extends React.Component {
       }
     });
     layout.run();
+    layout.pon('layoutstop').then(function( event ){
```
I think this can be combined with the other layoutstop function 8 lines up.

hmtinc replied:
It indeed can. The other one was just recently added by another PR. I have combined the two together

d2fong commented:
Also it might make sense to promote this code to the common/cy module.  Can you see if you can refactor it to be less coupled from the view?

hmtinc commented:
@d2fong I have refactored the search code and moved search to/common/cy. 
The module takes in a cytoscape object `cy` and a search query string, and applies the search styling to correct nodes, so it should be able to used elsewhere in the project now.
